src/hardware/pwm.rs

```diff
@@ -74,6 +74,13 @@ pub struct Pwm {
 }
 
 impl Pwm {
+    // PWM constants
+    const MAX_DUTY: u32 = 10000; // Maximum duty cycle valid for all channels.
+    const V_PWM: f32 = 3.3; // MCU PWM pin output high voltage
+    pub const MAX_CURRENT_LIMIT: f32 = ((Pwm::MAX_DUTY as f32 - 1.0) / Pwm::MAX_DUTY as f32)
```

Comment:
Just found [this](https://github.com/stm32-rs/stm32h7xx-hal/blob/8a76d7f9fdb938cde8ec5672c3becfc7c5254e10/src/pwm.rs#L1390-L1399) so I'll let the limit include 100% duty cycle again since we're not close to an overflow with 10000.